documentation/docs/libraries/python/examples/_09_transaction.mdx

```diff
@@ -0,0 +1,10 @@
+import CodeBlock from "@theme/CodeBlock";
+import transaction from "!!raw-loader!../../../../../bindings/python/examples/09_transaction.py";
+
+Sending value-based messages is a very straightforward process. You will only need to provide a valid `seed`, `outputs`
+(addresses and amounts) parameters to the
+[Client.message()](./../libraries/python/api_reference#messageseed-optional-account_index-optional-initial_address_index-optional-inputs-optional-input_range_begin-optional-input_range_end-optional-outputs-optional-dust_allowance_outputs-optional-index-optional-index_raw-optional-data-optional-data_str-optional-parents-optional-message)
+function. The method will find valid output(s) that can be used to fund the given amount(s) and the unspent amount will
+be sent to the same address.
```

Comment:
Maybe add something like this, so it's more clear that the funds go back to the sender address?
### 
```suggestion
function. The method will find valid output(s) that can be used to fund the given amount(s) and the remaining amount will
be sent back the input address.
```